### DISCUSSION TITLE
Facing issues while performing OCR using paddle on image files

### DISCUSSION THREAD

**Discussion by vnairergo** [PaddlePaddle/PaddleOCR#13494] [Q&A]
Hi Community,

I have been trying out paddleocr for a few implementations of ocr. My aim is to extract proper texts from standard documents (common format across a set of people). The trouble I am facing is that there is a lack of white space detection between words that I am noticing. For example if the document has a string : First Name - Abe Lincoln, the base model is detecting it as FirstName - AbeLincoln even though there is a visible separation between words. 

I am new to this community and paddle, thus I wanted some inputs on how can I avoid this problem. What pre-processing steps can I take to help resolve issues like these.

Thanks in advance.

**Comment by SWHL** [comment]
Could you provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)?
